### PR TITLE
wiki: add wiki for DEX trading account management

### PR DESCRIPTION
**Pull request by ukane-philemon**
This PR checks one of the boxes in https://github.com/decred/dcrdex/issues/1641.

[Click to Preview](https://github.com/ukane-philemon/dcrdex/wiki/DEX-Trading-Account-Management)

### CODE REVIEW

File: docs/wiki/DEX-Trading-Account-Management.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+## Adding an existing DEX Trading Account
+
+If you have an existing DEX account that you want to use with a new setup, here's how you can do it:
+
+1. Go to the `Settings` Page.
+2. Click on `Import Account`.
+
+    <img src="./images/dex-accounts-settings.png" width="300">
+
+3. Choose the exported DEX account `.json` file by clicking on `load from file`.
```
Comment (JoeGruffins) [inline docs/wiki/DEX-Trading-Account-Management.md:34]: Have input instructions but no corresponding export instructions. Looking at the dex settings page I think it's obvious but maybe a note here or should that be another `##` point?

Comment (ukane-philemon) [reply docs/wiki/DEX-Trading-Account-Management.md:34]: Thanks for the review @JoeGruffins. I'll include a guide on how to export.

File: docs/wiki/DEX-Trading-Account-Management.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+In this guide, we will discuss how you can manage your DEX Trading Account.
+
+Let's get started!
+
+## Setting Up Your DEX Trading Account
+
+Before you can start buying and selling on Decred DEX, you'll need a DEX trading
+account. If you've already followed all the steps in the [Initial Setup
+Guide](./Client-Installation-and-Configuration.md/#initial-setup), you're all
```
[the remark this answers is not in the document]
Comment (ukane-philemon) [reply docs/wiki/DEX-Trading-Account-Management.md:11]: > Clicking on this doesn't seem to show what is intended. Here I have pushed the changes to my wiki, please try clicking https://github.com/JoeGruffins/dcrdex/wiki/DEX-Trading-Account-Management

Yeah, thanks!

**Review by JoeGruffins — APPROVED** [review]
Looks good to me.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**